tmail-backend/combined-identity/src/main/java/com/linagora/tmail/combined/identity/UsersRepositoryModuleChooser.java

```diff
@@ -46,7 +46,7 @@ public class UsersRepositoryModuleChooser {
     public static List<Module> chooseModules(Implementation implementation) {
         return switch (implementation) {
             case LDAP -> ImmutableList.of(new LdapUsersRepositoryModule(), new CassandraDelegationStoreModule());
-            case COMBINED -> ImmutableList.of(new CombinedUsersRepositoryModule());
+            case COMBINED -> ImmutableList.of(new CombinedUsersRepositoryModule(), new CassandraDelegationStoreModule());
```

Comment:
if we add new CassandraDelegationStoreModule() all the time it's better to add it once directly in DistributedServer

I did this before because there was some bindings done (but maybe was not enough)